reviewing/dependencies-hierarchy/src/buildDependenciesHierarchy.ts

```diff
@@ -62,7 +63,9 @@ export async function buildDependenciesHierarchy (
 
   const result = {} as { [projectDir: string]: DependenciesHierarchy }
 
-  if (!currentLockfile) {
+  const lockfileToUse = maybeOpts.lockfileOnly ? wantedLockfile : currentLockfile
```

Comment:
I guess we can keep the name for the flag but as an option to buildDependenciesHierarchy it probably doesn't make sense as regardless of this boolean, a lockfile is read. Maybe it could be named `checkWantedLockfileOnly`.

Reply:
Good idea, thanks.